cases/8_Fake_Invoice_Scam/datagen.json

```diff
@@ -0,0 +1,332 @@
+    },
+    {
+      "topic": "Invoices",
+      "value": {
+        "InvoiceID": {
+          "_gen": "sequentialInteger"
+        },
+        "SupplierID": {
+          "_gen": "lookup",
+          "topic": "Suppliers",
+          "path": [
+            "value",
+            "SupplierID"
+          ]
+        },
+        "EmployeeApprovedID": {
+          "_gen": "lookup",
+          "topic": "Employees",
+          "path": [
+            "value",
+            "EmployeeID"
+          ]
+        },
+        "BalanceDue": {
+          "_gen": "uniformDistribution",
+          "bounds": [
+            50,
+            200
+          ],
+          "decimals": 2
+        },
+        "Currency": {
+          "_gen": "weightedOneOf",
+          "choices": [
+            {
+              "value": "$",
+              "weight": 17
+            },
+            {
+              "value": "£",
+              "weight": 1
+            },
+            {
+              "value": "€",
+              "weight": 2
+            }
+          ]
+        },
+        "DueDate": {
+          "_gen": "formatDateTime",
+          "ms": {
+            "_gen": "math",
+            "expr": "timestamp - offset",
+            "names": {
+              "timestamp": {
+                "_gen": "now"
+              },
+              "offset": {
+                "_gen": "uniformDistribution",
+                "bounds": [
+                  -604800000,
+                  15778476000
+                ],
+                "decimals": 0
+              }
+            }
+          },
+          "format": "yyyy-MM-dd"
+        }
+      ,
+        "SortCode": {
+          "_gen": "uniformDistribution",
+          "bounds": [
+            100000,
+            1000000
+          ],
+          "decimals": 0
+        },
+        "AccountNumber": {
+          "_gen": "uniformDistribution",
+          "bounds": [
+            10000000,
+            100000000
+          ],
+          "decimals": 0
+        },
+        "PaymentTime": {
+          "_gen": "formatDateTime",
+          "ms": {
+            "_gen": "math",
+            "expr": "timestamp - offset",
+            "names": {
+              "timestamp": {
+                "_gen": "now"
+              },
+              "offset": {
+                "_gen": "uniformDistribution",
+                "bounds": [
+                  846000000,
+                  15778476000
+                ],
+                "decimals": 0
+              }
+            }
+          },
+          "format": "yyyy-MM-dd HH:mm:ss.SSS"
+        },
+        "ItemExpected": {
+          "_gen": "string",
+          "expr": "#{Commerce.productName}"
+        },
+        "QuantityExpected": {
+          "_gen": "uniformDistribution",
+          "bounds": [
+            1,
+            500
+          ],
+          "decimals": 0
+        }
+      },
+      "localConfigs": {
+        "repeat": {
+          "rate": 0.015,
+          "times": 1
+        },
+        "events":{
```

Comment:
to make this more "real-time", consider a rate limit rather than event count bounds?